libnavigation-core/src/test/java/com/mapbox/navigation/core/replay/history/MapboxReplayerTest.kt

```diff
@@ -273,7 +273,7 @@ class MapboxReplayerTest {
 
         mapboxReplayer.playFirstLocation()
 
-        verify { replayEventsObserver wasNot Called }
+        verify(exactly = 0) { replayEventsObserver.replayEvents(any()) }
```

Comment:
changing to mutableSet causes hashcode to be called.

change this to verify the specific function was never called